jena-db/jena-tdb2/src/main/java/org/apache/jena/tdb2/store/nodetable/NodeTableCache.java

```diff
@@ -71,15 +71,17 @@
     private final Object        lock          = new Object();
     private volatile Thread     writingThread;
 
+    /** Build a node table cache, based on the node/nodIs setting in params. */
```

Comment:
s/nodIs/nodeIds? Not sure from the context here alone

Reply:
Done